scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -17,6 +17,13 @@ workflow GvsExtractCallset {
         String? fq_filter_set_site_table
         String? fq_filter_set_tranches_table
         String? filter_set_name
+
+        # if these are unset, default sensitivity levels will be used; if user is defining a cutoff, only use one type (sensitivity or lod)
+        Float? snps_truth_sensitivity_filter_level
+        Float? indels_truth_sensitivity_filter_level
+        Float? snps_lod_score_cutoff
```

Comment:
question: should we include lod cutoffs in the wdl? i feel like it's something most users won't want to use and might be more confusing than anything?

Reply:
I agree --let's remove the lod cutoffs from the WDL, it's not a production/public use case and if anything would be one of us or a methods dev running the tool locally.